hr_timesheet_name_customer/readme/USAGE.rst

```diff
@@ -0,0 +1,5 @@
+To use this module you need to:
+
+#. Go to Apps and install this module
```

Comment:
Modify "Description Customer" field value. Select timesheets you want to print add choose "Timesheets Customer" report.
"Description" column values will be populated from the "Description Customer" field.